src/containers/SearchPage/SelectMultipleFilter/SelectMultipleFilter.js

```diff
@@ -19,6 +19,7 @@ const GroupOfFieldCheckboxes = props => {
   const { id, className, name, options } = props;
   return (
     <fieldset className={className}>
+      <legend>{name}</legend>
```

Comment:
We can't add legend to fieldset as it changes the UI. It adds this:
<img width="341" height="326" alt="Screenshot 2025-08-19 at 14 15 37" src="https://github.com/user-attachments/assets/bc1efbe4-6a72-44a2-a3b7-4d5deb2abdaf" />
I.e. legend is a kind of label for fieldsets.

Reply:
Got it, removed!